v2.0/src/config/config.json

```diff
@@ -1,5 +1,5 @@
 {
-  "IS_LOCAL": false,
+  "IS_LOCAL": true,
```

Comment:
flip?

Reply:
aready done - I think you were looking at local code